resources/codegen_plan.toml

```diff
@@ -13,6 +13,16 @@ mode = "parse"
 source = "resources/codegen_inputs/colr.rs"
 target = "read-fonts/generated/generated_colr.rs"
 
+[[generate]]
+mode = "parse"
+source = "resources/codegen_inputs/cpal.rs"
+target = "read-fonts/generated/generated_cpal.rs"
+
+[[generate]]
+mode = "compile"
+source = "resources/codegen_inputs/cpal.rs"
+target = "write-fonts/generated/generated_cpal.rs"
+
```

Comment:
oh late note but we can leave this out until we're actually generating the `write-fonts` module.